python/cudf/cudf/core/column/column.py

```diff
@@ -217,6 +217,16 @@ def dropna(self, drop_nan: bool = False) -> ColumnBase:
         # The drop_nan argument is only used for numerical columns.
         return drop_nulls([self])[0]
 
+    def _dedup_preserve_order(self):
```

Comment:
Should this just be merged with the  `unique()` method? I'm thinking `col.unique()` returns unique values in arbitrary order, and `col.unique(preserve_order=True)` does  this.

Reply:
Discussed this offline and came to the conclusion that long term we should implement https://github.com/rapidsai/cudf/issues/11638 to back this.

Reply:
Done!